src/js/components/AppListComponent.jsx

```diff
@@ -98,16 +98,19 @@ var AppListComponent = React.createClass({
     var state = this.state;
     var appNodes = this.getAppNodes();
 
+    var pageIsLoading = state.fetchState === States.STATE_LOADING;
+    var noAppsOnPage = state.apps.length === 0;
```

Comment:
Definitely agree, this is a really nice readability enhancement. Pushed in f82662b.